scripts/generate.js

```diff
@@ -44,6 +53,43 @@ for (let component of components) {
   fs.writeFileSync(`${__dirname}/../build/${component.name}Icon.js`, fileContent);
 
   indexContent += `export ${component.name} from './${component.name}Icon';\n`;
+  
+  /*
+    Pushing the different formats of typescript definitions to their respective arrays.
+  */
+  defFileFillIns.declareIconClasses.push(`class ${component.name}Icon extends GenericMdiIcon {}`);
```

Comment:
I couldn't remember why, but there was a specific reason why it was done this way. So I tried to modify it and I immediately get errors.